.github/workflows/release.yml

```
@@ -7,14 +7,39 @@ on:
   push:
     tags:
       - 'v*'
+  pull_request:
+    types:
+  push:
+    branches: ["main"]
+      - closed
+    branches:
+      - main
```

sorry my previous suggestion wasn't quite on the right lines, should be:
```suggestion
  push:
    branches:
      - main
```